Monika After Story/game/script-songs.rpy

```diff
@@ -1191,6 +1191,37 @@ label mas_song_lamour_toujours:
     m 5ekbsa "I want nothing more than to be by your side forever, [player]~"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_god_knows",
+            category=[store.mas_songs.TYPE_SHORT],
+            prompt="God Knows",
+            random=True,
+            aff_range=(mas_aff.AFFECTIONATE,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_god_knows:
+    m 1hua "{i}~You know that{w=0.2}{nw}"
```

Comment:
Does the {i} statement here need to be closed with {/i}? That's the only thing I can think of that would make the build not pass, comparing it to #5828 which did pass. I'm not that versed in this kind of thing yet, but maybe it doesn't hurt to try?

Reply:
Open {i} statements are fine, they terminate on new line. It fails because of missing expressions. Check the travis log, it lists any problems a pr has.

Reply:
Alright